### PR TITLE
[RBAC/Roles] Apply Default Roles

### PR DESCRIPTION
**Pull request by kirmmin**
`!account create` now includes an optional 3rd parameter of role for the user. If omitted, it will use the default role as specific in the WorldServer config, or failing that "1".

### CODE REVIEW

**Comment by AsterBunnymund** [comment]
!account create

File: Source/NexusForever.WorldServer/Command/Handler/AccountCommandCategory.cs [hunk 1]
```diff
@@ -14,16 +15,21 @@ public class AccountCommandCategory : CommandCategory
...
+            [Parameter("Role")]
+            uint? role = null)
         {
             if (DatabaseManager.Instance.AuthDatabase.AccountExists(email))
             {
                 context.SendMessage("Account with that username already exists. Please try another.");
                 return;
             }
 
+            if (role == null)
```
Comment (Rawaho) [inline Source/NexusForever.WorldServer/Command/Handler/AccountCommandCategory.cs]: Null coalescing assignment operator is your friend here :D

Comment (kirmmin) [reply Source/NexusForever.WorldServer/Command/Handler/AccountCommandCategory.cs]: ??=  ? ;)

Comment (MaxtorCoder) [reply Source/NexusForever.WorldServer/Command/Handler/AccountCommandCategory.cs]: yessir

Comment (kirmmin) [reply Source/NexusForever.WorldServer/Command/Handler/AccountCommandCategory.cs]: Operator is now in use.

**Pull request merged**